src/pool/srv_pool.c

```diff
@@ -822,6 +780,59 @@ pool_svc_name_cb(daos_iov_t *id, char **name)
 	return 0;
 }
 
+static int
+pool_svc_load_uuid_cb(daos_iov_t *id, uuid_t db_uuid)
+{
+	char   *path;
+	int	rc;
+
+	if (id->iov_len != sizeof(uuid_t))
+		return -DER_INVAL;
+	path = pool_svc_rdb_uuid_path(id->iov_buf);
+	if (path == NULL)
+		return -DER_NOMEM;
+	rc = uuid_load(path, db_uuid);
+	D_FREE(path);
```

Comment:
Just a minor suggestion...it might make sense to have a function to pair with pool_svc_rdb_uuid_path so we don't have a D_FREE here that doesn't have an alloc in the immediate scope.